modules/libcom/src/error/errlog.c

```diff
@@ -466,17 +467,38 @@ int eltc(int yesno)
 {
     errlogInit(0);
     epicsMutexMustLock(pvt.msgQueueLock);
-    pvt.toConsole = yesno;
+    pvt.toConsole = yesno ? 1 : 0;
```

Comment:
Wouldn't it be clearer to do a cast to `bool` here or use `!!yesno`?

Were you fixing a warning with this change? From what I can see, all usage of `eltc` is with 0 or 1 already.